### PR TITLE
Update staff list in `Beatmap_Spotlights`

### PR DESCRIPTION
**Pull request by Hiviexd**
`Curation System` needs a rework since things have changed internally, will work on that after the upcoming season starts

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Review by ZeroPyrozen — CHANGES_REQUESTED** [review]
### Add these flags reference

`en.md`
```
[flag_TN]: /wiki/shared/flag/TN.gif "Tunisia"
[flag_MY]: /wiki/shared/flag/MY.gif "Malaysia"
```

`fr.md`
```
[flag_TN]: /wiki/shared/flag/TN.gif "Tunisie"
[flag_MY]: /wiki/shared/flag/MY.gif "Malaisie"
```

`id.md`
```
[flag_TN]: /wiki/shared/flag/TN.gif "Tunisia"
[flag_MY]: /wiki/shared/flag/MY.gif "Malaysia"
```

`zh.md`
```
[flag_AT]: /wiki/shared/flag/AT.gif "奥地利"
[flag_AR]: /wiki/shared/flag/AR.gif "阿根廷"
[flag_TN]: /wiki/shared/flag/TN.gif "突尼斯"
[flag_KR]: /wiki/shared/flag/KR.gif "韩国"
[flag_MY]: /wiki/shared/flag/MY.gif "马来西亚"
```

### Remove these flags reference

`en.md`
```
[flag_ES]: /wiki/shared/flag/ES.gif "Spain"
[flag_RU]: /wiki/shared/flag/RU.gif "Russian Federation"
[flag_SG]: /wiki/shared/flag/SG.gif "Singapore"
```

`fr.md`
```
[flag_ES]: /wiki/shared/flag/ES.gif "Espagne"
[flag_RU]: /wiki/shared/flag/RU.gif "Fédération de Russie"
[flag_SG]: /wiki/shared/flag/SG.gif "Singapour"
```

`id.md`
```
[flag_ES]: /wiki/shared/flag/ES.gif "Spanyol"
[flag_RU]: /wiki/shared/flag/RU.gif "Rusia"
[flag_SG]: /wiki/shared/flag/SG.gif "Singapura"
```

`zh.md`
```
[flag_ES]: /wiki/shared/flag/ES.gif "西班牙"
[flag_SG]: /wiki/shared/flag/SG.gif "新加坡"
```

**Comment by Walavouchey** [comment]
it's better to just link to https://megaapplepi.github.io/flag-wiki-osu/ instead of putting everything in a huge comment, since it's faster to use the tool

**Comment by ZeroPyrozen** [comment]
Ah I didn't know that tool exist, thanks. I will use it next time

**Review by cl8n — APPROVED** [review]

File: wiki/Beatmap_Spotlights/en.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ The Beatmap Spotlights project is run by various community members across all ga
 | Role | Members |
 | :-- | :-- |
 | Project lead | ![][flag_PL] [Venix](https://osu.ppy.sh/users/5999631) |
-| Project manager | ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) |
+| Project manager | ![][flag_TN] [Hivie](https://osu.ppy.sh/users/14102976) |
```
Comment (cl8n) [inline wiki/Beatmap_Spotlights/en.md:25]: if you update this article again, I think it'd be good to explain what the difference between these two roles are. or pick better words to get it across

**Pull request merged**